packages/react/src/components/DatePicker/DatePicker.Skeleton.js

```diff
@@ -37,7 +44,14 @@ const DatePickerSkeleton = ({ range, id }) => {
   return (
     <div className={`${prefix}--form-item`}>
       <div
-        className={`${prefix}--date-picker ${prefix}--date-picker--short ${prefix}--date-picker--simple ${prefix}--skeleton`}>
+        className={cx(
+          `${prefix}--date-picker`,
+          `${prefix}--date-picker--short`,
+          `${prefix}--date-picker--simple`,
+          `${prefix}--skeleton`,
+          className
```

Comment:
```suggestion
          [className]: className
```

Reply:
Is this being passed in as part of the arguments list instead of as an object?